### PR TITLE
test(schema): add json schema test script and fix classes schema

### PR DESCRIPTION
**Pull request by Buluphont**
A few things happen here:
- Move schema from `data/` to a new `test/schema/` folder
- Fix table subschema (rows were incorrectly defined as an array of Entry,
    when in reality they are actually an array of array of Entry
- Splice the Entry subschema into its own schema
- Add a test script `alltests.js` to `test/`

NOTE: the test script is currently NOT added to the test pipeline, because
Giddy's changes to classes.js are not finished, so his updated classes.json being
merged in is blocked. When it's merged in, we should add an `&& node ./test/alltest.js`
to the `package.json`'s test script.

TL;DR:
To test this, please: 
1. drop in the latest `classes.json` to `./data/`
2. append `&& node ./test/alltest.js` to `package.json`'s `test` task
3. `npm test`

### CODE REVIEW

**Comment by TheGiddyLimit** [comment]
@Buluphont u a real one
